browser/ui/webui/ai_chat/ai_chat_ui_page_handler.cc

```diff
@@ -431,19 +442,20 @@ void AIChatUIPageHandler::GetFaviconImageData(
       &favicon_task_tracker_);
 }
 
-absl::optional<mojom::SiteInfo> AIChatUIPageHandler::BuildSiteInfo() {
-  if (active_chat_tab_helper_ && active_chat_tab_helper_->HasPageContent() ==
-                                     PageContentAssociation::HAS_CONTENT) {
-    mojom::SiteInfo site_info;
-    site_info.title =
-        base::UTF16ToUTF8(active_chat_tab_helper_->web_contents()->GetTitle());
-    site_info.is_content_truncated =
-        active_chat_tab_helper_->IsPageContentsTruncated();
-
-    return site_info;
-  }
-
-  return absl::nullopt;
+mojom::SiteInfo AIChatUIPageHandler::BuildSiteInfo() {
+  bool has_content = active_chat_tab_helper_->HasPageContent() ==
+                     PageContentAssociation::HAS_CONTENT;
+  mojom::SiteInfo site_info;
+  site_info.title =
+      has_content ? base::UTF16ToUTF8(
+                        active_chat_tab_helper_->web_contents()->GetTitle())
+                  : std::string();
```

Comment:
should this be `if (has_content) { site_info.title = ... }` to avoid empty string and have optional value?

Reply:
done